app/home/home.css

```diff
@@ -1,10 +1,5 @@
 
-.vertical-center {  
-  min-height: 100vh; 
-  display: flex;
-  align-items: center;
-}
-.header{
+/*.header{
```

Comment:
ok, thank you, the problem is that sometimes do some tests which maybe, can use in the future